corehq/apps/geospatial/static/geospatial/js/gps_capture.js

```diff
@@ -142,5 +142,31 @@ hqDefine("geospatial/js/gps_capture",[
 
     $(function () {
         $("#no-gps-list").koApplyBindings(dataItemListModel());
+
+        var initMap = function (centerCoordinates) {
+            'use strict';
+
+            var self = {};
```

Comment:
In light of removing `var map`, we can also remove `self` since we are currently not using it. This can always be added back in if we ever do need to return `self` in the future.

Reply:
Cool. Seems reasonable to remove that.
Removed that and also moved the function out.
https://github.com/dimagi/commcare-hq/pull/33467/commits/c4a8369eaf85cf835814c113d913ba9e56e0ffd5